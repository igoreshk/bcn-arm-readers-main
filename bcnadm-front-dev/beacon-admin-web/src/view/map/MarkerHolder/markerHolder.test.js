import React from 'react';
import { shallow } from 'enzyme';
import { MarkerHolder } from './index';

const props = {
  markerHandler: {},
  vertexDtoToMarker: jest.fn(),
  getDtos: jest.fn(),
  setMarkerListeners: jest.fn(),
  onMarkerCreate: jest.fn(),
  onRemoveAllMarkers: jest.fn(),
  changeEditMode: jest.fn(),
  levelNumber: 1,
  selectedMapProviderName: 'Google',
  selectedMapProvider: {},
  selectedMap: {},
  match: {
    params: { level: '60952c00d9fc0118256dc408', layer: 'scaling' }
  }
};

const wrapper = shallow(<MarkerHolder {...props} />);

describe('MarkerHolder', () => {
  it('should match snapshot', () => expect(wrapper).toMatchSnapshot());
});
