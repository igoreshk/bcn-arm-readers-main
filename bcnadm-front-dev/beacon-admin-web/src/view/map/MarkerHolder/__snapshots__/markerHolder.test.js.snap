// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarkerHolder should match snapshot 1`] = `
<LocalizedComponent
  changeEditMode={[MockFunction]}
  handleRemoveMarkers={[Function]}
  levelNumber={1}
  selectedMapProviderName="Google"
/>
`;
